src/Compilers/CSharp/Test/CommandLine/CommandLineTests.cs

```diff
@@ -8394,11 +8420,13 @@ private void VerifyQuotedInvalid<T>(string name, string value, T expected, Func<
             [Fact]
```

Comment:
I am not sure this is the only check we need. I didn't test this but I believe it'll still not work because COM support in Mono on Windows is not really working.